### PR TITLE
Update large_use_case

### PR DESCRIPTION
**Pull request by AnnikaLau**

### CODE REVIEW

File: docs/models/icon/usage.md [hunk 1]
```diff
@@ -109,14 +130,18 @@ git clone --depth 1 --recurse-submodules --shallow-submodules -b ${SPACK_TAG} ht
...
 
 # Build ICON
-cd /path/to/icon
+cd /path/to/icon-build-folder
 spack env activate -d config/cscs/spack/${SPACK_TAG}/todi_gpu_nvhpc
 spack install
 ```
 
 
+### Sanits
```
Comment (mjaehn) [inline docs/models/icon/usage.md:139]:
```suggestion
### Santis
```

**Review by mjaehn — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: docs/models/icon/usage.md (hunk 1 above)
Comment (AnnikaLau) [reply docs/models/icon/usage.md:139]: I don't see any difference?

**Comment by AnnikaLau** [comment]
@mjaehn please approve!

**Review by mjaehn — APPROVED** [review]

**Pull request merged**